src/part/social-bar.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Locale } from "src/i18n/locale";
```

Comment:
I think we do not need to import Locale here since it is not used

Reply:
@protyagov oh, I forgot that I wanted to ask, probably we should add alt for links?

Reply:
Right, would be good to add ALT links -- they help Google when they index things.

Reply:
I meant title, not alt

Reply:
Right, something like this:
`alt={_("FOOTER.APPLE_BUTTON.ALT")} `